src/test/js/log-karaoke/noStages.js

```diff
@@ -90,7 +90,7 @@ module.exports = {
             this
                 .elements('css selector', 'div.result-item.success', function (collection2) {
                     const count2 = collection2.value.length;
-                    this.assert.notEqual(count, count2);
```

Comment:
I would write this assert as
`this.assert.ok(count !== count2, 'counts should be different')`